bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -38,6 +38,9 @@
 
 	private ImageButton currPaint;
 
+	EditText annotatedTextBox;
```

Comment:
I think it's good to keep in at the class level since we use it in two different methods (one is onCreate, the other is a clickListener)